src/components/DateRangePicker.jsx

```diff
@@ -282,15 +283,15 @@ class DateRangePicker extends React.Component {
     if (withPortal || withFullScreenPortal) {
       return (
         <Portal>
-          {this.renderDayPicker()}
+          {this.renderDayContentsPicker()}
```

Comment:
isn't this still rendering a day picker?

Reply:
Whoops! D: find and replace :X